voltdbclient/url.go

```diff
@@ -25,9 +26,23 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
-		if u.Port() == "" {
+		if getPort(u.Host) == "" {
 			u.Host = u.Host + ":21212"
 		}
 	}
 	return u, nil
 }
+
+func getPort(host string) string{
```

Comment:
@mehmetalisavas thanks for the change, however I find the wording not descriptive enough. So I suggest you add the following comment instead.

```go
// This function body was taken from the go standard library url.Port() method
// for the sake of maintaining backward compatibility with older version of Go
// Which falls under the following copyright notice
//
// Copyright 2009 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file https://golang.org/LICENSE
//
// [Link to the source] https://golang.org/pkg/net/url/#URL.Port
//
// TODO (@gernest) : Replace this with another implementation!
```

I will eventually replace this but not any time soon.